.drone.yml

```diff
@@ -11,7 +11,17 @@ notify:
     on_started: false
     on_success: false
     webhook_url: $$slack_webhook
+publish:
+  npm:
+    email: $$npm_drone_email
+    password: $$npm_drone_password
+    username: $$npm_drone_username
+    when:
+      branch: master
 script:
 - sudo pip install -q git+https://$REPORT_CARD_GITHUB_REPO_TOKEN@github.com/Clever/report-card.git; GITHUB_API_TOKEN=$REPORT_CARD_GITHUB_STATUS_TOKEN report-card --publish || true
 - npm install
 - npm test
+- make build # publish
+- nvm install 0.10.29
+- nvm use 0.10.29
```

Comment:
Well, maybe, but this is how we do publishing with node 0.10 right now. You can see other repos as an example. It gets cleaned up with node 6.